packages/coinbase-wallet/src/index.ts

```diff
@@ -77,28 +77,23 @@ export class CoinbaseWallet extends Connector {
   public async connectEagerly(): Promise<void> {
     const cancelActivation = this.actions.startActivation()
 
-    await this.isomorphicInitialize()
+    try {
+      await this.isomorphicInitialize()
+
+      if (!this.connected) throw new Error('No existing connection')
```

Comment:
type definition says `connectEagerly` fails silently. https://github.com/Uniswap/web3-react/blob/main/packages/types/src/index.ts#L125.

Should the comment in the type be updated?